api-tester/build.gradle

```diff
@@ -28,6 +48,9 @@ android {
 dependencies {
     implementation project(path: ':purchases')
     implementation project(path: ':feature:amazon')
+    implementation project(path: ':debugview')
 
-    implementation libs.androidx.annotation
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.8.0"))
```

Comment:
can this be in the toml file?

Reply:
Oh right, I was running into some issues when moving it to the toml so I left it here... Will give it another try to see if I can figure it out

Reply:
Done in https://github.com/RevenueCat/purchases-android/pull/1075/commits/819508fc522f2ad2a187aed677d0e65b2b5919ff